content/gallery.md

```diff
@@ -6,34 +6,38 @@ date: 2020-03-25
 {{< gallery >}}
 {{< figure link="/sites/default/files/Mizu-AppManager.png" caption="Applications Manager" >}}
 {{< figure link="/sites/default/files/Mizu-Search.png" caption="File Search" >}}
+{{< figure link="/sites/default/files/Mizu-Desktop.png" caption="Mizu Theme" >}}
```

Comment:
If this is the only page those images are used, it might make sense to have them in a `gallery` subfolder, so it is easier to find / edit them for the next person?